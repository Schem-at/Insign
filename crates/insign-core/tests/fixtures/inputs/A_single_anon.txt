@rc([0,0,0],[3,2,1])
#doc.label="Patch A"
#doc.type="test"
