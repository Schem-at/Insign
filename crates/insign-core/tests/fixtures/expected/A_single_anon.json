{
  "__anon_0_0": {
    "bounding_boxes": [
      [
        [
          0,
          0,
          0
        ],
        [
          3,
          2,
          1
        ]
      ]
    ],
    "metadata": {
      "doc.label": "Patch A",
      "doc.type": "test"
    }
  }
}